Common/DTO/UserInformation.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace DTO
+{
+    public class UserInformation
```

Comment:
I got confused about all this different names. UserInformation, UserInfoViewModel, and a lot of others. Can we make them more similar?

Reply:
> I got confused about all this different names. UserInformation, UserInfoViewModel, and a lot of others. Can we make them more similar?
https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93viewmodel

Reply:
Okay, but I meant mostly names: How about UserInformation -> UserInformationViewModel.

Reply:
I see that in this way: view models are used with the gui itself, and models just serve a purpose of representing some data about some object

Reply:
Okay